src/test/javascript/spec/component/course/course-exercises.component.spec.ts

```diff
@@ -182,7 +182,7 @@ describe('CourseExercisesComponent', () => {
         component.course!.exercises![component.course!.exercises!.length] = new ModelingExercise(UMLDiagramType.ClassDiagram, course, undefined) as Exercise;
 
         component.activeFilters.clear();
-        component.activeFilters.add(ExerciseFilter.OVERDUE);
+        component.activeFilters.add(ExerciseFilter.OVERDUE).add(ExerciseFilter.UNRELEASED);
```

Comment:
You need to test how it affects the filtered result because you've changed that as well.